Automattic/jetpack
# Update/boost Use Admin_UI package to register top level menu

This PR makes use of the Admin UI package (unmerged) to register the Boost menu under the Jetpack top level menu 

#### Changes proposed in this Pull Request:

* Use admin_ui package to add admin menu

#### Jetpack product discussion


pbuNQi-1w3-p2#comment-2639

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Activate Boost
* Check that the menu item is under the Jetpack top level menu
* Try it alongside Jetpack and without Jetpack. And also alongside the Backup plugin. Try all possible combinations
* Also try with Jetpack the plugin being connected and disconnected.
